<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="my_SaveButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Device</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="my_OnlyKeyboardCheck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Only Keyboard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="my_OnlyPointerCheck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Only Pointer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="my_DisableOnStartup">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable device on startup</string>
    </property>
   </widget>
   <widget class="QComboBox" name="my_DeviceListCbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>561</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="my_ToggleButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Device</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="my_NoMessage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable popup message</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
